### DISCUSSION TITLE
Favicon not displayed

### DISCUSSION THREAD

**Discussion by ArthurBlanchon** [fuma-nama/fumadocs#1123] [Q&A]
Hi, 

I can't figure out where to setup the favicon. 
I'm used to just having the favicon.ico file in the public folder but this doesn't seem to work.

Does Fumadocs requiere a specific setup ?

**Comment by fuma-nama** [comment]
I think Next.js only picks the icon.png or favicon.ico in app folder, Fumadocs doesn't modify metadata object

**Reply by ArthurBlanchon** [reply]
Thank you for the quick response.

I fixed the issue using an `icon.svg` file in the /app folder. ✅
I also tested : 
- `icon.svg` in /app/docs. It worked for /docs pages ✅
- `icon.svg` in /app/(home) It worked for the / page ✅
- `icon.ico` in /app/docs. It didn't work ❌
- `icon.ico` in /app/(home). It didn't work ❌
- `icon.ico` in /app. It didn't work ❌
- `favicon.ico` in /app. It didn't work ❌
- `favicon.ico` in /public. It didn't work ❌

I seem to have an issue with `.ico` files. I can't investigate more. 
I would be curious to know if others are having the same issue.

Thanks a lot @fuma-nama for the project. 
It's awesome ❤️.

**Reply by iamdanusabu** [reply]
This works

**Reply by Sma1lboy** [reply]
Just for an update to avoid others having the same issue like me, I did check the fumadocs/app/docs and realized `icon.png` also works well, there are a lot of issues converting logos from png to svg :(

> The fumadocs repository is the best example that teaches us how to use fumadocs XD

**Reply by fuma-nama** [reply]
shouldn't be hard since it's a basic thing in Next.js (https://nextjs.org/docs/app/api-reference/file-conventions/metadata/app-icons)
